src/ApiService/ApiService/onefuzzlib/WebhookOperations.cs

```diff
@@ -269,7 +269,7 @@ public WebhookMessageLogOperations(ILogger<WebhookMessageLogOperations> log, IOn
             }
             return sendResult.IsOk;
         } catch (Exception exc) {
-            _logTracer.LogError(exc, "Send Webhook");
+            _logTracer.LogError("Send Webhook: {exception}", exc);
```

Comment:
Does the retry code eventually throw an exception if we exhaust retries?

Reply:
No, we just log a warning:

_logTracer.LogWarning("sending webhook event failed, re-queued {WebhookId}:{EventId}", newMessage.WebhookId, newMessage.EventId);

Reply:
Good point though, we can log an exception so alerting picks this up